components/raftstore/src/store/peer.rs

```diff
@@ -1569,19 +1569,28 @@ where
         self.raft_group.snap()
     }
 
-    fn add_ready_metric(&self, ready: &Ready, metrics: &mut RaftReadyMetrics) {
-        metrics.message += ready.messages().len() as u64;
-        metrics.commit += ready.committed_entries().len() as u64;
-        metrics.append += ready.entries().len() as u64;
+    fn add_ready_metric(&self, ready: &Ready, metrics: &mut RaftMetrics) {
```

Comment:
Why the change?

Reply:
Don't want to introduce import from metrics as `RaftReadyMetrics` is deleted now.